skin/src/main/java/org/researchstack/skin/AppPrefs.java

```diff
@@ -24,6 +23,11 @@
         prefs = PreferenceManager.getDefaultSharedPreferences(context);
     }
 
+    public static void init(Context context) {
+        instance = new AppPrefs(context);
+    }
+
+    @Deprecated
     public static synchronized AppPrefs getInstance(Context context)
```

Comment:
Nit-pick: We could remove synchronized to make it cleare that synchronize isn't needed or properly implemented and it won't affect the method signature for backwards compatibility purposes.